README.md

```diff
@@ -152,7 +156,10 @@ Now you'll need to do steps (2) everytime you make a changes and want to build t
 **Note:**
```

Comment:
Let's keep all the Notes here (including the `pipenv` one) and only link it above in order to not repeat ourselves! :smile: